phantomgames/Home/HomeScreenViewController.swift

```diff
@@ -11,8 +11,8 @@ class HomeScreenViewController: UIViewController{
     
     // MARK: IBOutlets
     
-    @IBOutlet weak var HomeCollectionView: UICollectionView!
-    @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak private var HomeCollectionView: UICollectionView!
```

Comment:
why is this variable starting with a capital letter?
Please read through the recommended standards that we have spend 4 lessons on and made documents for.